docs/design-tech/billing.md

```diff
@@ -73,9 +73,13 @@ We listen for two particular events:
 
 Before this flow is completable we'll have to pre-register our [prices](https://stripe.com/docs/api/prices) with stripe.
 
+## Billing States
+
+Our current implementation is simple. A user is either subscribed or not. When a user misses a payment or chooses to cancel their subscription, their data cap is immediately reduced to 1 MB.
+
 ## Testing
 
-TODO
+A stripe integration can only be tested if provided a [secret testing API key](https://stripe.com/docs/keys#obtain-api-keys). [`stripe_cli`](https://stripe.com/docs/stripe-cli) also provides a way to emulate webhook events locally.  
```

Comment:
Add instructions regarding testing for google play

Reply:
I have added basic instructions for testing on google play? Maybe your diff is old again.